### PR TITLE
[KOGITO-2564] Add Red Hat headers to all sources in Kogito apps

### PR DESCRIPTION
**Pull request by bxf12315**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by bxf12315** [comment]
@cristianonicolai  please review it

File: pom.xml [hunk 1]
```diff
@@ -65,6 +65,26 @@
         </plugin>
       </plugins>
     </pluginManagement>
+    <plugins>
+      <plugin>
+        <groupId>com.mycila</groupId>
```
Comment (cristianonicolai) [inline pom.xml:70]: shouldnt config be inherent from parent, the one thefined in kogito-runtimes?

File: src/resources/META-INF/services/com.mycila.maven.plugin.license.PropertiesProvider [hunk 2]
```diff
@@ -0,0 +1,3 @@
+com.mycila.maven.plugin.license.git.CopyrightRangeProvider
```
Comment (cristianonicolai) [inline src/resources/META-INF/services/com.mycila.maven.plugin.license.PropertiesProvider]: this should be inherited from parent pom configuration

**Comment by cristianonicolai** [comment]
closing as outdated

**Pull request closed without merging**